pkg/service/utgen/injector.go

```diff
@@ -54,12 +97,13 @@ func (i *Injector) libraryInstalled() ([]string, error) {
 		return i.extractPackageNames(out), nil
 
 	case "typescript", "javascript":
-		cmd := exec.Command("sh", "-c", "npm list --depth=0 --parseable | sed 's|.*/||'")
+		cmd := exec.Command("sh", "-c", "npm list --depth=0")
+		cmd.Dir = "/home/ahmed/Desktop/Work/Task3/samples-typescript/express-mongoose"
 		out, err := cmd.Output()
 		if err != nil {
 			return nil, fmt.Errorf("failed to get JavaScript/TypeScript dependencies: %w", err)
 		}
-		return extractString(out), nil
+		return i.extractJavaScriptPackageNamesWithVersions(out), nil
```

Comment:
why only version for javascript do the same for other languages, also please find a more suitable name for function, this seems to be too long